ghcide/src/Development/IDE/GHC/Compat.hs

```diff
@@ -241,9 +238,9 @@ nonDetOccEnvElts = occEnvElts
 
 type ModIfaceAnnotation = Annotation
 
-#if MIN_VERSION_ghc(9,3,0)
-nameEnvElts :: NameEnv a -> [a]
-nameEnvElts = nonDetNameEnvElts
+#if !MIN_VERSION_ghc(9,3,0)
+nonDetNameEnvElts :: NameEnv a -> [a]
+nonDetNameEnvElts = nameEnvElts
```

Comment:
This name flip warrants explanation:
Since this function is named `nonDetNameEnvElts` in ghc 9.4+
I flipped the "compat" function to this new name and aliased the old function with this new name.
With the expectation that the compat function can be dropped when when we remove ghc 9.2 as opposed to keeping the old name even when it's no longer needed.